### ISSUE TITLE
Tacx App does not pair with ANT FE-C trainer

### ISSUE THREAD

**Issue by WouterJD** [totalreverse/ttyT1941#13]
Hi @totalreverse 

I got this message from a FortiusANT user
`I was trying to explain that the Tacx App (on Windows) sees the FortiusAnt as a FE-C trainer in ANT but will not connect to it. Tacx support has explained to me that only bluetooth connections are possible on computers (windows or Mac). And they also said their smartphone app (iOS or android) will only work with their trainers-no others. So the only option to connect is connect to the Tacx app via bluetooth with a computer.`

My question to you:
Should TTS connect to an ANT FE-C trainer? And if so, -but since not pairing with FortiusANT- would there be an unknown/undocumented requirement so that TTS allows pairing?

Note that TrainerRoad, Zwift and Rouvy connect faultlessly to FortiusANT....

**Comment by totalreverse** [comment]
There are so many apps and programs from Tacx. 

Tacx Fortius Software
TTS (Tacx Trainer Software) 1 to 4  
TCA (Tacx cycline app)
TTA and TDA (Tacx Training App and Tacx Desktop App)

I do not know the newer TCA , TTA and TDA. Probably the FortiusANT user you quote is using the Tacx Desktop App. I often read, that the newer Tacx software products do not support ANT+ to control the trainer. You can only connect your CAD and SPEED and HR-Monitor via ANT+,  thats all. But I cannot confirm this information through  own tests. 

You have tested your FortiusANT with the TTS4 software?

**Comment by WouterJD** [comment]
It appears that bluetooth is the preferred interface and that ANT+ is left for legacy; Tacx and Rouvy go for BT.

I tried TTS4 and it did not want to pair and -when I tested- I concluded this might be another close-out of non-tacx devices so did not spend time on it, where Rouvy offered what was looking for.

So the challenge would be to make a BT-interface...
Let's see whether that's something for next winter...

Thanks again for thinking with me.
